quodlibet/config.py

```diff
@@ -189,6 +189,15 @@
         # (0 = disabled i.e. arithmetic mean)
         "bayesian_rating_factor": "0.0",
 
+        # This is used to decrease the high and increase the small weights, which are
+        # used in the calculation of numeric function averages (e.g. ~#rating:avg).
+        # This is done, so that longer tracks have a higher impact on the average than
+        # shorter ones. For example, when calculating the average rating of an album, we
+        # want this to be the case. However, without smoothing, short tracks had nearly
+        # no effect on the average, while long ones would dominate the average.
+        # If this is -1, averages are unweighted. If this is 0, no smoothing is applied.
+        "weight_smoothing_factor": "10",
```

Comment:
`weight_bias` just came to mind as well... `smoothing` still makes me think of time series or other 2D data somehow

Reply:
Maybe it's slightly pedantic, but `weight_bias` seems incomplete to me, as it doesn't at all reference the fact that weights are compressed / smoothed. But maybe `weight_smoothing_bias`.. hmm

I hadn't actually done this until now, but when I read the [Wikipedia article](https://en.wikipedia.org/wiki/Smoothing) for smoothing, the term actually does seem somewhat appropriate. 

> In statistics and image processing, to smooth a data set is to create an approximating function that attempts to capture important patterns in the data, while leaving out noise or other fine-scale structures/rapid phenomena. In smoothing, the data points of a signal are modified so individual points higher than the adjacent points (presumably because of noise) are reduced, and points that are lower than the adjacent points are increased leading to a smoother signal.

I guess one difference is that we're going beyond adjacent points.

> smoothing still makes me think of time series or other 2D data somehow

That's technically correct though, as the data is two dimensional (track_index and it's weight, for example: `[0.7, 0.8, 0.2]`).

Reply:
But we're never talking about rating track 1.25, so kinda different perhaps (like you say the adjacency thing). Anyway...

Reply:
We could also call it `communism_factor` :laughing: